index.js

```diff
@@ -111,21 +119,28 @@ app.get('/movies/directors/:director', passport.authenticate('jwt', { session: f
 });
 
 // Get all users
-app.get('/users', passport.authenticate('jwt', { session: false }), (req, res) => {
-    Users.find()
-        .then((users) => {
-            res.status(201).json(users);
-        })
-        .catch((err) => {
-            console.error(err);
-            res.status(500).send('Error: ' + err);
-        });
-});
+// There should never be a route like this in your app
+// app.get('/users', passport.authenticate('jwt', { session: false }), (req, res) => {
```

Comment:
For playing around sure you can do this but for a production and secure code you should not have a route like this.